python/tank/platform/engine.py

```diff
@@ -2898,3 +2915,23 @@ def __pick_environment(engine_name, tk, context):
 
     return env_name
 
+def _get_command_prefix(properties):
+    """
+    If multiple commands are registered with the same name, attempt to construct a unique
+    prefix from other information in the command's properties dictionary to distinguish one
+    command from another. Uses the properties' ``app`` and/or ``group`` keys to create the
+    prefix.
+
+    :param dict properties: Arbitrary key/value information related to a registered command.
+    :returns: A unique identifier for the command as a str.
+    """
+    prefix = ""
```

Comment:
nitpicky - maybe this can be done in a more pythonic way with a `join`?

```python
prefix = []
if properties.get("app"):
  prefix.append(properties["app"].instance_name)
if properties.get("group"):
  prefix.apend(properties["group"])
return ":".join(prefix)
```